lib/pivotal-tracker-api/service.rb

```diff
@@ -147,57 +154,84 @@ def stories(opts={})
-        
+
         api_url = "/projects/#{project_id}/stories"
         api_url.append_pivotal_params(opts[:parameters])
-        
+
         puts "\n****** URL: #{api_url}\n\n"
-        
+
         response = PivotalAPI::Client.get(api_url)
         json_story = JSON.parse(response, {:symbolize_names => true})
         PivotalAPI::Stories.from_json(json_story)
       end
-      
+
       def story(opts={})
         # opts:
         #  :project_id - REQUIRED - A valid pivotal project ID
         #  :story_id - REQUIRED - A valid pivotal story ID
         #  :fields - OPTIONAL - specific fields to ask pivotal to return for each story
-        
+
         raise ArgumentError.new("missing required key/value :project_id") unless opts[:project_id]
         raise ArgumentError.new("missing required key/value :story_id") unless opts[:story_id]
-        
+
         project_id = opts[:project_id]
         story_id = opts[:story_id]
         opts[:parameters] = {} unless opts[:parameters]
         opts[:parameters][:fields] = opts[:fields] if opts[:fields] && opts[:parameters][:fields].nil?
-        
+
         api_url = "/projects/#{project_id}/stories/#{story_id}"
         api_url.append_pivotal_params(opts[:parameters])
-        
+
         puts "\n****** URL: #{api_url}\n\n"
-        
+
         response = PivotalAPI::Client.get(api_url)
-        json_story = JSON.parse(response, {:symbolize_names => true})
-        PivotalAPI::Story.from_json(json_story)
+        load_story_from_response(response)
       end
 
       def update_story(story_id, project_id, updates={})
         raise ArgumentError.new("missing required parameter project_id") unless project_id
         raise ArgumentError.new("missing required parameter sotry_id") unless story_id
-        
+
         api_url = "/projects/#{project_id}/stories/#{story_id}"
         PivotalAPI::Client.put(api_url, updates)
       end
 
+      def create_story(project_id, attrs={})
+        raise ArgumentError.new("missing required parameter project_id") unless project_id
+
+        api_url = "/projects/#{project_id}/stories"
+        response = PivotalAPI::Client.post(api_url, attrs)
+        load_story_from_response(response)
+      end
+
+      def create_comment(story_id, project_id, text)
+        raise ArgumentError.new("missing required parameter project_id") unless project_id
+        raise ArgumentError.new("missing required parameter sotry_id") unless story_id
+        raise ArgumentError.new("missing required parameter text") unless text
+
+        if text.size > 20000
+          raise ArgumentError.new("Text must be less than 20000") 
+        end
+
+        api_url = "/projects/#{project_id}/stories/#{story_id}/comments"
+        response = PivotalAPI::Client.post(api_url, text: text)
+        json_story = JSON.parse(response, {:symbolize_names => true})
+        PivotalAPI::Comment.from_json(json_story)
+      end
+
+      private
+        def load_story_from_response(response)
```

Comment:
Maybe just name this `story_from_response`? Don't really think it needs to be prefixed with `load_`.